docs/DeveloperGuide.md

```diff
@@ -283,16 +283,49 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 ### Use cases
 
-(For all use cases below, the **System** is the `AddressBook` and the **Actor** is the `user`, unless specified otherwise)
+(For all use cases below, the **System** is the `HobbyList` and the **Actor** is the `user`, unless specified otherwise)
 
-**Use case: Delete a person**
+**Use case: Add an activity**
 
 **MSS**
 
-1.  User requests to list persons
-2.  AddressBook shows a list of persons
-3.  User requests to delete a specific person in the list
-4.  AddressBook deletes the person
+1. User requests to add an activity to the list
+2. HobbyList adds that activity to the list and shows a confirmed message
+
+    Use case ends.
+
+**Extensions**
+
+* 1a. The activity is already existed.
```

Comment:
I think this sentence might sound a bit weird?
Maybe "The activity already exists."?